build_tools/update_torch_ods.sh

```diff
@@ -42,8 +42,3 @@ if [ ! -z ${TORCH_MLIR_EXT_MODULES} ]; then
 fi
 
 set +u
```

Comment:
As tempting as it is to just delete these lines and call it done, I'm not sure we can do this: people still use this tool like this.